Runtimes/Core/cmake/caches/Vendors/Apple/Darwin.toolchain.cmake

```diff
@@ -0,0 +1,116 @@
+# Toolchain configuration when compiling for Darwin Platforms
+set(CMAKE_SYSTEM_NAME Darwin)
+set(CMAKE_Swift_COMPILER_WORKS YES)
+set(CMAKE_C_COMPILER_WORKS YES)
+set(CMAKE_CXX_COMPILER_WORKS YES)
+
+find_program(XCRUN_EXECUTABLE NAMES "xcrun" REQUIRED)
+
+if(NOT CMAKE_OSX_SYSROOT)
+  if(DEFINED ENV{SDKROOT})
+    set(sdk_name $ENV{SDKROOT})
+  else()
+    message(FATAL_ERROR "SDKROOT environment variable not set")
+  endif()
```

Comment:
This might be easier to read as:

```cmake
if(NOT DEFINED ENV{SDKROOT})
  message(FATAL_ERROR "The 'SDKROOT' environment variable is not set")
endif()

execute_process(COMMAND "${XCRUN_EXECUTABLE}" --show-sdk-path -sdk $ENV{SDKROOT} ...)
```

Reply:
cleaned up the check